src/Pages/Main/Components/LimitedEvent/LimitedEvent.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import ProductCard from '../ProductCard/ProductCard';
+import './LimitedEvent.scss';
+
+class LimitedEvent extends Component {
+  constructor() {
+    super();
+    this.state = {
+      time: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setTimer();
+  }
+
+  componentWillUnmount() {
+    this.setTimer();
+  }
+
+  setTimer = () => {
+    const x = setInterval(() => {
+      let countdownDate = new Date('Apr 24, 2021, 00:00:00').getTime();
+      let now = new Date().getTime();
+      let remaining = countdownDate - now;
+      let hours = Math.floor(
+        (remaining % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)
+      );
+      let minutes = Math.floor((remaining % (1000 * 60 * 60)) / (1000 * 60));
+      let seconds = Math.floor((remaining % (1000 * 60)) / 1000);
+      const twoDigits = time => {
+        return (time < 10 ? '0' : '') + time;
+      };
+      this.setState({
+        time: `${twoDigits(hours)}:${twoDigits(minutes)}:${twoDigits(seconds)}`,
+      });
+      if (remaining < 0) {
+        clearInterval(x);
+        return;
+      }
```

Comment:
```suggestion
      let seconds = Math.floor((remaining % (MILLISECONDS * SECONDS)) / MILLISECONDS);
      
      const twoDigits = time => (time < 10 ? '0' : '') + time;

      this.setState({
        time: `${twoDigits(hours)}:${twoDigits(minutes)}:${twoDigits(seconds)}`,
      });
      
      if (remaining < 0) {
        clearInterval(x);
        return;
      }
```
- 상수 변수로 표현력을 높여줄 수 있는 부분이 있으면 적극적으로 활용해주세요!
```js
const MILLISECONDS = 1000
const SECONDS = 60
const MINUTES = 60
const HOURS = 24
```
- 한줄로 작성할 수 있는 부분은 한줄로 작성하면 좋겠습니다
- 로직 잘 작성해주셨네요! 👍

Reply:
감사합니다 수정하였습니다!